assembly/gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Tue Mar 27 15:55:39 UTC 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
```

Comment:
Any reason not to go straight to the current version 6.2.2?

Reply:
I was thinking that using an older version might give us some insights about the minimum gradle version that is supported. (However, you are right that version 5.1.1 is a bit arbitrary.)

Reply:
If it as an effect on the usability within a customer Gradle build, the older the version the better. It is not a good habit to push our customers to use the latest Gradle only to run the compliance tooling.

Reply:
It has no effect on the customer. This is just the Gradle version used to build the Gradle plugin.

Reply:
Probably, as long as the Gradle plugin only uses the stable Gradle API, it will be compatible with multiple Gradle versions, even older ones - and I think that our plugin only uses a rather small part of the API.

However, if my investigations of the Kotlin incompatibility are correct, the dependencies the Antenna plugin drags in restrict the Gradle version that can be used. So because of our dependency to OkHttpClient and (transitive) Kotlin 1.3, the plugin will not be compatible with Gradle 4.x.

Reply:
Ah, I was indeed mistaken that the Gradle version used for building the plugin would have no influence on the consumer, because of the `gradleApi()` call [here](https://github.com/eclipse/antenna/blob/1502d6ef07abcae23c9179c5189cb06708fe5b62/assembly/gradle-plugin/build.gradle#L15) (i.e. the plugin is built against the API of the Gradle version used for building). I somehow thought we'd build the Gradle plugin with pure Maven (as explained [here](https://stackoverflow.com/a/27556212/1127485)), but in that case we wouldn't need the Gradle wrapper at all anyway.

Reply:
We once had a bigger issue when we updated the Maven dependency and got feedback on projects using quite old versions. But moving towards Gradle 5.1.1 as base version sounds okish, especially when we have good reason like a technical dependency.

Reply:
Any objection then @blaumeiser-at-bosch if I also move the Gradle version used for our dependencies from 4.1 to 5.1.1 to align?

Reply:
Please do so

At @bs-ondem: We should keep this somewhere, so that we point that out in the release notes.